notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1294 @@
+{
```

Comment:
<p>Encapsulate this in an if statement, so the code only tries to create a bucket if it doesn't exist. This is cleaner.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done.</p>